src/pages/management/configuration-api/v3.5/index.mdx

```diff
@@ -3189,6 +3189,55 @@ curl -X POST \
 
 </Section>
 
+<Section>
+
+<Text>
+
+### Get Product Source
+
+This method retrieves the "source" parameters that were passed when activating the LiveChat product.
+
+#### Specifics
+
+|                 |                                                                            |
+| --------------- | -------------------------------------------------------------------------- |
+| HTTP Method     | **GET**                                                                    |
+| Method URL      | `https://api.livechatinc.com/v3.5/configuration/action/get_product_source` |
+| Required scopes | -                                                                          |
+
+#### Request
+
+This method accepts no parameters
+
+#### Response
+
+| Field         | Notes                                                                      |
+| ------------- | -------------------------------------------------------------------------- |
```

Comment:
In the below 3, I am not sure what you exactly mean, so please take a look at my propositions as I believe they may be more clear from the language perspective:)

Reply:
This is to say that setting those params is only done once, when creating the LiveChat license, i.e. "activating the LiveChat product" using the Global Accounts speak, though that functionality is not publicly available/documented. I like your proposal 👌